cpp/random_walk_module/random_walk_module.cpp

```diff
@@ -0,0 +1,81 @@
+#include <mg_procedure.h>
+#include <ctime>
+#include <mg_exceptions.hpp>
+#include <mg_utils.hpp>
+#include <random>
+#include <vector>
+
+namespace {
+
+constexpr char const *kProcedureGet = "get";
+constexpr char const *kFieldPath = "path";
+
+constexpr char const *kArgumentStart = "start";
+constexpr char const *kArgumentLength = "length";
+
+/// Memgraph query module implementation of a random walk algorithm.
+/// Random walk is an algorithm that provides random paths in a graph.
+///
+/// @param args Memgraph module arguments
+/// @param graph Memgraph graph instance
+/// @param result Memgraph result storage
+/// @param memory Memgraph memory storage
+void Generate(const mgp_list *args, const mgp_graph *graph, mgp_result *result, mgp_memory *memory) {
+  try {
+    auto start = mgp_value_get_vertex(mgp_list_at(args, 0));
+    auto length = mgp_value_get_int(mgp_list_at(args, 1));
+
+    auto path = mgp_path_make_with_start(start, memory);
+    auto vertex = start;
+    srand(time(NULL));
```

Comment:
I guess this shouldn't be in the release?

Reply:
Sorry, forgot about this PR for some time. 
I am probably missing something obvious so could you please elaborate? :smile:

Reply:
This module was made a while ago and would need an update to match the new API. The commented line has nothing to do with the question, sorry :sweat_smile:

Reply:
I get it now. :slightly_smiling_face: 
I will update it to the new API.